src/FS.Embedded/build/netstandard2.0/Microsoft.Extensions.FileProviders.Embedded.targets

```diff
@@ -1,66 +1,80 @@
 <Project>
 
   <PropertyGroup>
-    <TaskFolder Condition=" '$(MSBuildRuntimeType)' == 'Core' ">netstandard1.5</TaskFolder>
-    <TaskFolder Condition=" '$(MSBuildRuntimeType)' != 'Core' ">net461</TaskFolder>
-    <TaskAssembly>$(MSBuildThisFileDirectory)..\..\tasks\$(TaskFolder)\Microsoft.Extensions.FileProviders.Embedded.Manifest.Task.dll</TaskAssembly>
+    <_FileProviderTaskFolder Condition="'$(MSBuildRuntimeType)' == 'Core'">netstandard1.5</_FileProviderTaskFolder>
+    <_FileProviderTaskFolder Condition="'$(MSBuildRuntimeType)' != 'Core'">net461</_FileProviderTaskFolder>
+    <_FileProviderTaskAssembly>$(MSBuildThisFileDirectory)..\..\tasks\$(_FileProviderTaskFolder)\Microsoft.Extensions.FileProviders.Embedded.Manifest.Task.dll</_FileProviderTaskAssembly>
   </PropertyGroup>
 
   <UsingTask
-    AssemblyFile="$(TaskAssembly)"
+    AssemblyFile="$(_FileProviderTaskAssembly)"
     TaskName="Microsoft.Extensions.FileProviders.Embedded.Manifest.Task.GenerateEmbeddedResourcesManifest" />
 
   <PropertyGroup>
-    <GenerateEmbeddedFilesManifest Condition="'$(GenerateEmbeddedFilesManifest)' == ''">false</GenerateEmbeddedFilesManifest>
-    <EmbeddedFilesManifestFileName Condition="'$(EmbeddedFilesManifestFileName)' == ''">Microsoft.Extensions.FileProviders.Embedded.Manifest.xml</EmbeddedFilesManifestFileName>
+    <PrepareResourceNamesDependsOn>_CalculateEmbeddedFilesManifestInputs;$(PrepareResourceNamesDependsOn)</PrepareResourceNamesDependsOn>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(GenerateEmbeddedFilesManifest)' == 'true'">
```

Comment:
Doing this is problematic since we may not generate the file. I moved it to a separate target that runs before resx generation that only adds it if there are any manifest files of interest.